lsio-plex.json

```diff
@@ -0,0 +1,46 @@
+{
+    "LSIO-Plex": {
+        "description": "Plex media server",
+        "ui": {
+            "slug": "web"
+        },
+        "volume_add_support": true,
+        "website": "https://plex.tv/",
+        "version": "24.09.2015",
+        "containers": {
+            "plex": {
+                "image": "linuxserver/plex",
+                "launch_order": 1,
+                "opts": [
+                    "--net=host",
+                    ""
+                ],
+                "environment": {
+                    "PUID": { 
+                        "description": "User ID Plex will run as",
+                        "label": "PUID"
+                    },
+                    "PGID": {
+                        "description": "Group ID Plex will run as",
+                        "label": "PGID"
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Plex configuration. Eg: create a Share called plex-config for this purpose alone.",
+                        "label": "Config Storage"
+                    },
+                    "/transcode": {
+                        "optional": true,
+                        "description": "Transcode directory to offload heavy writes in a docker container",
+                        "lable": "Transcode Location"
```

Comment:
@pecigonzalo "lable" should be "label". I'm still relatively new to Git so I figure you're probably better at making the change than I am.